src/OpenLoco/Windows/BuildVehicle.cpp

```diff
@@ -764,8 +764,7 @@ namespace OpenLoco::Ui::Windows::BuildVehicle
         const auto* company = CompanyManager::get(CompanyManager::getControllingId());
         if (!company->isVehicleIndexUnlocked(item))
         {
-            Error::open(StringIds::cant_build_pop_5_string_id, StringIds::vehicle_is_locked);
-            return;
+            GameCommands::setErrorTitle(StringIds::vehicle_is_locked);
```

Comment:
The title would be cant_build_pop_5_string_id. You don't need any of this branch anymore.

Reply:
That doesn't display as nicely since it doesn't list the vehicle that you're trying to build.

Reply:
Your suggestion:
![image](https://user-images.githubusercontent.com/7483209/156996158-7f5294a3-bfc5-44cf-8cf6-67120e7e1a57.png)

My version:
![image](https://user-images.githubusercontent.com/7483209/156996347-3d90b113-bf55-4663-8ec9-720435d511a0.png)